src/safeds_runner/server/_memoization_map.py

```diff
@@ -82,6 +332,25 @@ def __str__(self) -> str:
         )
 
 
+# Lambda = Stat Key Extractor, Boolean = Reverse Order
```

Comment:
I don't know if this is the best way to organize strategies, but it was the easiest for now.
I didn't use a class hierarchy (abstract base class + implementations for strategies), as they would just return the lambda and the direction (which would be a bit pointless in my opinion).